cmp/form/dropdown/BaseDropdownField.js

```diff
@@ -17,6 +18,11 @@ import {HoistField} from '../HoistField';
  */
 export class BaseDropdownField extends HoistField {
 
+    static propTypes = {
+        /** Text to display when control is empty */
+        placeholder: PT.string
```

Comment:
E.g. these dropdown fields also could take a width, right?  (To make the point that I don't think we need it on the others, not suggesting we add it here.)